.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,53 @@
+# CI stages to execute against Pull Requests
+name: Robottelo - CI
```

Comment:
Your copy-paste is showing :smile:

Reply:
Description does say it's implemented to be like robottelo ;)